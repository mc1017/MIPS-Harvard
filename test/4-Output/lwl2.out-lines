CPU : OUT :4009754624
