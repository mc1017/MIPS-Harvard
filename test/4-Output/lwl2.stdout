CPU : OUT :4009754624
TB : Finished : active=0
test/1-ArithmeticandLS/lwl2.v:32: $finish called at 28 (1s)
